message_structure.md

```diff
@@ -39,6 +39,40 @@ Aligning with [RFC 6648](https://tools.ietf.org/html/rfc6648.html), DIDComm expl
+
+Because a DIDComm encrypted message is also a JWE, a true but less specific media type MIGHT be `application/jose`. (Although `application/jwt` is a registered media type, `application/jwe` and `application/jws` [are not](https://tools.ietf.org/html/rfc7516#section-9). This is [deliberate](https://mailarchive.ietf.org/arch/msg/jose/FRTPwiOLOc5DILfY_QFZyemn9VU/).) Using the more generic type is not recommended, as content categorized in this way is unlikely to get the DIDComm-specific handling it needs. Similarly, it is also true but overly generic and therefore not recommended to describe a DIDComm encrypted message as `application/json`.
+
+When persisted as a file or attached as a payload in other contexts, the file extension for DIDComm encrypted messages SHOULD be `dcem`, giving a globbing pattern of `*.dcem`; this SHOULD be be read as "Star Dot D C E M" or as "D C E M" files.
+
+The name of this file format is "DIDComm Encrypted Message." We imagine people will say, "I am looking at a DIDComm Encrypted Message", or "This file is in DIDComm Encrypted Message format", or "Does my editor have a DIDComm Encrypted Message plugin?" A recommended icon for this file format depicts an envelope protected by a wax seal ([svg](collateral/dcem.svg) | [256x256](collateral/dcem-256.png) | [128x128](collateral/dcem-128.png) | [64x64](collateral/dcem-64.png)):
+
+![DIDComm Encrypted Message Icon](collateral/dcem-128.png)
+
+### DIDComm Plaintext Messages
+
+A DIDComm message in its plaintext form, not packaged into a protective JWE envelope, is known as a DIDComm Plaintext Message. Plaintext messages lack confidentiality and integrity guarantees, and are therefore not normally transported across security boundaries. However, this may be an appropriate format to inspect in debuggers, and it is the format used in this spec to give examples of headers and other internals. Depending on ambient security, plaintext may or may not be an appropriate format for DIDComm data at rest.
+
+When higher-level protocols are built atop DIDComm, applications remove the protective envelope and process the plaintext that's inside. Specs for such protocols typically document message structure and provide examples in this format; the encrypted envelope is assumed but ignored as a low-level detail.
+
+Applications running one or more DIDComm-based protocols may wish to define their own media types for the protocols they support. Absent such customization, the preferred media type for a generic DIDComm plaintext message SHOULD be `application/didcomm-plaintext+json`. This correctly conveys the fact that code handling such content functions at a different level from DIDComm's internals, and that generic JSON tools and actions are likely to be a helpful fallback in processing the content.
```

Comment:
I would like to propose the use of the following notation for the mime types
application/didcomm.plaintext+json
application/didcomm.encrypted+json
application/didcomm.signed+json

I'm less opinionated on the language of `plaintext`, `encrypted` and `signed`.

Based on mime type notes at https://en.wikipedia.org/wiki/Media_type